packages/ui/primitives/document-flow/add-subject.tsx

```diff
@@ -29,27 +46,38 @@ export type AddSubjectFormProps = {
 
 export const AddSubjectFormPartial = ({
   documentFlow,
-  recipients: _recipients,
+  recipients: recipients,
   fields: _fields,
   document,
   numberOfSteps,
   onSubmit,
 }: AddSubjectFormProps) => {
   const {
+    control,
     register,
     handleSubmit,
     formState: { errors, isSubmitting },
+    getValues,
+    setValue,
   } = useForm<TAddSubjectFormSchema>({
     defaultValues: {
-      email: {
+      meta: {
         subject: document.documentMeta?.subject ?? '',
         message: document.documentMeta?.message ?? '',
+        timezone: document.documentMeta?.timezone ?? DEFAULT_DOCUMENT_TIME_ZONE,
+        dateFormat: document.documentMeta?.dateFormat ?? DEFAULT_DOCUMENT_DATE_FORMAT,
       },
     },
   });
 
   const onFormSubmit = handleSubmit(onSubmit);
 
+  const hasDateField = _fields.find((field) => field.type === 'DATE');
```

Comment:
If we're now using `_fields` we can rename the symbol to `fields`, we use `_var` for unused variables that we want to keep in scope.

Reply:
🫡